### PR TITLE
Add a file listing ressources about writing

### PR DESCRIPTION
**Pull request by genintho**

### CODE REVIEW

File: university/writing.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Step up your Writing game at work
```
Comment (veganstraightedge) [inline university/writing.md:1]:
```suggestion
# Step up your writing game at work
```

IMO, `writing` isn't a proper now here. Nor is the whole heading in Title Case.

File: university/writing.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+
+- Practical advice to improve the quality of your pull request description
+https://www.youtube.com/watch?v=BbIILUSmSk4
```
Comment (veganstraightedge) [inline university/writing.md:4]:
```suggestion
- [Video: Practical advice to improve the quality of your pull request description](https://www.youtube.com/watch?v=BbIILUSmSk4)
```

File: university/writing.md (hunk 1 above)
Comment (genintho) [reply university/writing.md]: I liked the emphasis :)

Comment (veganstraightedge) [reply university/writing.md]: Could we use HTML's mechanism for this?

`_writing_`

File: university/writing.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
+# Step up your writing game at work
```
Comment (veganstraightedge) [inline university/writing.md:1]:
```suggestion
# Step up your _writing_ game at work
```

Comment (veganstraightedge) [reply university/writing.md:1]: If you want.

**Review by veganstraightedge — APPROVED** [review]

**Pull request merged**